src/vue/forms/TfaForm.vue

```diff
@@ -50,6 +50,7 @@
       <clipboard-field
         :value="factor.secret"
         :label="'tfa-form.secret-lbl' | globalize"
+        tooltip-type="tfaClipboard"
```

Comment:
Let's try using `tooltip.js`. When you be in the office let's discuss this situation.